index.d.ts

```diff
@@ -110,6 +95,27 @@ interface RequestContext {
    * specified in {@link ConfigParams.identityClaimFilter identityClaimFilter} removed.
    */
   user?: object;
+
+  /**
+   * Fetches the OIDC userinfo response.
+   *
+   * ```js
+   * app.use(auth());
+   *
+   * app.get('/user-info', async (req, res) => {
+   *   const userInfo = await req.oidc.fetchUserInfo();
+   *   res.json(userInfo);
+   * })
+   * ```
+   *
+   * @param options Options for the UserInfo request.
+   */
+  fetchUserInfo(options?: {
+    verb?: 'GET' | 'POST';
+    via?: 'header' | 'body' | 'query';
+    tokenType?: string;
+    params?: object;
+  }): Promise<UserinfoResponse>;
```

Comment:
my recommendation: don't expose any of these options. The default is the REQUIRED to implement for all providers that support the userinfo endpoint.